bcda/cclf/testutils/cclfUtils.go

```diff
@@ -63,7 +64,8 @@ func ImportCCLFPackage(acoSize, environment string, fileType models.CCLFFileType
 	switch environment {
 	case
 		"test",
-		"test-new-beneficiaries":
+		"test-new-beneficiaries",
+		"preadj":
```

Comment:
to be consistent should this be something like `test-preadj`?

Reply:
yeah thats a good thought, I can make that change